src/m365/spo/commands/term/term-get.ts

```diff
@@ -160,11 +160,8 @@ class SpoTermGetCommand extends SpoCommand {
         }
 
         if (terms._Child_Items_.length > 1) {
-          const disambiguationText = terms._Child_Items_.map(c => {
-            return `- ${this.getTermId(c.Id)} - ${c.PathOfTerm}`;
-          }).join(os.EOL);
-
-          throw new Error(`Multiple terms with the specific term name found. Please disambiguate:${os.EOL}${disambiguationText}`);
+          const resultAsKeyValuePair = formatting.convertArrayToHashTable('Id', terms._Child_Items_);
+          terms._Child_Items_[0] = await Cli.handleMultipleResultsFound<Term>(`Multiple terms with the specific term name found.`, resultAsKeyValuePair);
```

Comment:
Hmm, odd one this, let's keep it like this. This file would need some refactoring I think.

Reply:
ye, you mean the `_Child_Items_` 😅. Also don't like this but also had the same impression not to touch to many areas with a single PR which I guess we already do 🙂